DangoPlop/Assets/Scripts/Ball_Spawn.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Ball_Spawn : MonoBehaviour {
+
+    public GameObject ball;
+    public GameObject spawnPos;
+    public GameObject spawnPos2;
+    private int random;
+    public int maxBalls;
+    public float spawnWait;
+    public static int count;
+    public static int smallDeathCount;
+    private int targetScore;
+    private bool notInLoop;
+
+    // Use this for initialization
+    void Start () {
+        count = 0;
+        smallDeathCount = 0;
+        StartCoroutine(SpawnWaves());
+        targetScore = 1000;
+        notInLoop = false;
+    }
+	
+	// Update is called once per frame
+	void Update () {
+        if (count < maxBalls && notInLoop)
+        {
+            StartCoroutine(SpawnWaves());
+        }
+
+        if (ScoreManager.Score >= targetScore)
```

Comment:
ScoreManager isnt a variable, its a class. I accessed the Score variable this way since it was a static variable